app/src/main/java/com/example/sharingang/utils/UIViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.sharingang.utils
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class UIViewModel : ViewModel() {
```

Comment:
This is a really vaguely named class: "UIViewModel" could apply to any of our ViewModels: all of them are about the UI.

(It would also be a good idea to document this class; let's get into the habit of doing that)

Reply:
True, well I thought we could use this to keep specific UI elements in memory, like the spinners selected positions. Since it could also be used for other classes, this is the more general name I could come up with. Anything you would recommend instead?

Reply:
OrderingViewModel?

Reply:
Perfect